### PR TITLE
given a vpc this queries the name servers to check they are functional

### PR DESCRIPTION
**Pull request by sstarcher**

### CODE REVIEW

**Comment by sstarcher** [comment]
let me know if you need anything

**Comment by mattyjones** [comment]
@sstarcher  Can you rebase so I can pull please.

Thanks

**Comment by sstarcher** [comment]
@mattyjones Rebased for you.  I'll rebase the other pull request when this one is merged.

File: bin/check-dhcp-dns.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#! /usr/bin/env ruby
+#
+# check-ebs-snapshots
```
Comment (tas50) [inline bin/check-dhcp-dns.rb]: wrong name here

**Comment by tas50** [comment]
This is a really handy check and it's something I'll probably pull into my environment.  The name is rather confusing though.  It seems like it should be check-vpc-nameservers.  It is pulling the DHCP option set, but it's really just checking the name servers for a particular VPC.

**Comment by tas50** [comment]
Also make sure to add the plugin to the readme.

**Comment by sstarcher** [comment]
@tas50 that's a much better name. Fixed and added to readme

**Comment by sstarcher** [comment]
I believe I have the commit not as polluted as it was...
Let me know if I got everything.

**Comment by mattyjones** [comment]
@sstarcher  Can you do a rebase and they we will merge

Thanks!

**Pull request merged**